packages/@uppy/aws-s3-multipart/src/index.js

```diff
@@ -34,6 +34,8 @@ function assertServerError (res) {
 }
 
 module.exports = class AwsS3Multipart extends Plugin {
+  static VERSION = require('../package.json').version
```

Comment:
Is there an advantage to this vs setting `this.version` in the constructor? Cleaner this way, at the top?

Reply:
tbh I didn't consider putting it on this.version. my idea was to have a property `AwsS3Multipart.VERSION = version`, but then i saw that we do `module.exports = class XYZ` in most places, and i didn't want to refactor to something like
```js
class XYZ {}
XYZ.VERSION = version
module.exports = XYZ
```
so I used the static fields babel plugin instead

we could do this.version, but the version is conceptually more a property of the plugin (`Tus.VERSION`) rather than the instance (`getPlugin('Tus').version`), i think

Reply:
>  more a property of the plugin, rather than the instance

Yep, sure, makes sense, forgot that it would be on an instance.